CHANGES

```diff
@@ -14,6 +14,8 @@
       in 3.4.0. This ended up being a bad idea as two separate connection
       pools be considered equal yet manage a completely separate set of
       connections.
+    * HSET command now can accept multiple pairs. HMSET has been marked as 
+      deprecated now. Thanks to @laixintao #1271
```

Comment:
@laixintao this unfortunately doesn't mention that it requires at least Redis 4.0.0. We got bitten by this when updating `redis-py` (from 3.4.1 to 3.5.2) in a system that runs Redis 3.2.4.

@andymccurdy I couldn't find any information about which version(s) of Redis is supported by `redis-py`. Did I miss something?

Reply:
@ala-abelton This is completely backwards compatible with older versions of Redis server. You can still use the `key` and `value` that were present in all previous versions of redis-py. We could add a comment to the function that `mapping` requires Redis server 4.0 or greater.

redis-py works with all known versions of Redis server. Obviously you need a version of Redis server that supports the commands and command options you're working with.

Reply:
@andymccurdy sorry, I misread the diff. It seems that our issue didn't come from `redis-py`, but from https://github.com/rq/rq. Thanks for the quick reply!